.github/workflows/ci.yml

```diff
@@ -90,20 +90,29 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      - name: Determine tag
+      - name: Determine Docker Tag
         uses: haya14busa/action-cond@v1
         id: imagetag
         with:
           cond: ${{ github.event_name == 'pull_request' }}
           if_true: ${{ github.sha }}
-          if_false: "master"
+          if_false: 'master'
 
-      - name: Build
-        uses: docker/build-push-action@v1
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: crazy-max/ghaction-docker-buildx@v1
         with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: dexidp/dex
-          tags: ${{ steps.imagetag.outputs.value }}
-          add_git_labels: true
-          push: ${{ github.event_name == 'push' }}
+          buildx-version: latest
+
+      - name: Login to GitHub Docker Registry
```

Comment:
The Name says it is GitHub Docker registry, but I think this pushes to Docker Hub.

Reply:
This has now been addressed.